examples/website-demos/social-messaging/src/assets/NewChat.tsx

```diff
@@ -0,0 +1,35 @@
+import { useViewContext } from "../contexts/ViewContext";
+
+export const NewChat: React.FC = () => {
+    const { isNewChat, setNewChat } = useViewContext();
+
+    return (
+        <svg
+            width="40"
+            height="40"
+            viewBox="0 0 40 40"
+            fill="none"
+            onClick={() => {
+                if (!isNewChat) {
+                    setNewChat(true);
+                }
+            }}
+            style={{ borderRadius: '20px', cursor: 'pointer', marginRight: '8px' }}
+            xmlns="http://www.w3.org/2000/svg">
+            <path
+                d="M0 20C0 8.95431 8.95431 0 20 0C31.0457 0 40 8.95431 40 20C40 31.0457 31.0457 40 20 40C8.95431 40 0 31.0457 0 20Z"
+                fill="#005FFF"
```

Comment:
confirming none of these fills need to be variables?

Reply:
updated!